src/database/migrations/versions/2da0dc469be8_temp_migration.py

```
@@ -5,48 +5,67 @@
-               type_=sa.String(length=64),
-               existing_nullable=False)
-    op.alter_column('refresh_tokens', 'token',
-               existing_type=sa.VARCHAR(length=255),
-               type_=sa.String(length=64),
-               existing_nullable=False)
+    op.alter_column(
+        "activation_tokens",
+        "token",
+        existing_type=sa.VARCHAR(length=255),
+        type_=sa.String(length=64),
+        existing_nullable=False,
+    )
+    op.alter_column(
+        "password_reset_tokens",
+        "token",
+        existing_type=sa.VARCHAR(length=255),
+        type_=sa.String(length=64),
+        existing_nullable=False,
+    )
+    op.alter_column(
+        "refresh_tokens",
+        "token",
+        existing_type=sa.VARCHAR(length=255),
+        type_=sa.String(length=64),
+        existing_nullable=False,
+    )
     # ### end Alembic commands ###
 
 
 def downgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
-    op.alter_column('refresh_tokens', 'token',
-               existing_type=sa.String(length=64),
-               type_=sa.VARCHAR(length=255),
-               existing_nullable=False)
-    op.alter_column('password_reset_tokens', 'token',
-               existing_type=sa.String(length=64),
-               type_=sa.VARCHAR(length=255),
-               existing_nullable=False)
-    op.alter_column('activation_tokens', 'token',
-               existing_type=sa.String(length=64),
-               type_=sa.VARCHAR(length=255),
-               existing_nullable=False)
+    op.alter_column(
+        "refresh_tokens",
+        "token",
+        existing_type=sa.String(length=64),
+        type_=sa.VARCHAR(length=255),
+        existing_nullable=False,
+    )
+    op.alter_column(
+        "password_reset_tokens",
+        "token",
+        existing_type=sa.String(length=64),
+        type_=sa.VARCHAR(length=255),
+        existing_nullable=False,
+    )
```

When decreasing/increasing column lengths in migrations, it's safer to add explicit comments or checks in the migration (e.g., assert no value length > 64) to avoid silent failures or data truncation. Consider adding such checks before altering columns, or perform a data-migration to adjust token values.